Pod/Classes/VOKCoreDataCollectionTypes.h

```diff
@@ -0,0 +1,61 @@
+//
+//  VOKCoreDataCollectionTypes.h
+//  Pods
+//
+//  Created by Carl Hill-Popper on 10/23/15.
+//
+//
+
+#ifndef VOKCoreDataCollectionTypes_h
+#define VOKCoreDataCollectionTypes_h
+
+#if __has_feature(objc_generics)
+//forward declare classes used here since we don't know where this will be included
+@class NSManagedObject;
+@class NSManagedObjectID;
+@class VOKManagedObjectMap;
+
+//include the pointer for this one since the fallback is id
+#define VOKManagedObjectSubclassPtr __kindof NSManagedObject *
```

Comment:
`__kindof` tells the compiler that subclasses of the type are valid.  The previous method signature where this was used returned `id` before; I kept it the same for the fallback version.  I agree that it should probably be `NSManagedObject *` though. I'll change it.